#!/usr/bin/env python
PACKAGE = "egocylindrical"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("diameter", int_t, 0, "Diameter of pixel neighborhood", 5,0,21)
gen.add("sigmaColor", double_t, 0, "Filter sigma in the color space", 10, 0, 150)
gen.add("sigmaSpace", double_t, 0, "Filter sigma in the coordinate space", 10, 0, 150)


exit(gen.generate(PACKAGE, "egocylindrical", "Filter"))
